base/process.jl

```diff
@@ -647,6 +647,9 @@ Run a command object, constructed with backticks. Throws an error if anything go
 including the process exiting with a non-zero status.
 """
 function run(cmds::AbstractCmd, args...)
+    if isempty(cmds.exec) # Issue 19094
```

Comment:
Ah okay, I saw it in other files every now and then so I thought it'd be some undocumented convention. Removed it, should be explained by the message it throws